### PR TITLE
Iteration 2 Complete 

### PR DESCRIPTION
**Pull request by ghost**
@zacisaacson @freeheeling @mcat56 Iteration 2 is complete! We have refactored to make sure the code looks clean the only area that will still have seduo code. are methods that will be written in the next iteration

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
```
Comment (mcat56) [inline .gitignore]: We may want to take these out?

File: Rakefile [hunk 2]
```diff
@@ -1,7 +1,15 @@
+<<<<<<< HEAD
```
Comment (mcat56) [inline Rakefile]: To be taken out!

**Review by mcat56 — APPROVED** [review]
Little clean up otherwise looks good!

**Review by freeheeling — APPROVED** [review]
Approved pending resolution of merge conflicts in .gitignore and Rakefile

**Review by ghost — COMMENTED** [review]
I have corrected the two files that do not match so everything else was approved by the team. I will go ahead and merge

**Pull request merged**